migration/migrator/migrations/course/20210204140110_modify_subsection.py

```diff
@@ -0,0 +1,3 @@
+
+def up(config, database, semester, course):
+    database.execute('ALTER TABLE users ALTER COLUMN registration_subsection SET DEFAULT '' NOT NULL')
```

Comment:
You probably need to do the following sequence to deal with any existing null values in the DB:
```suggestion
    database.execute("UPDATE users SET registration_subsection = '' WHERE registration_subsection IS NULL")
    database.execute('ALTER TABLE users ALTER COLUMN registration_subsection SET DEFAULT '' NOT NULL')
```

Reply:
Should include a down migration as well to reverse the column default / not null change. Don't need to include an update though to reverse `''` into nulls as they're interchangeable in the system.